tests/benchdnn/dnnl_common.cpp

```diff
@@ -849,6 +849,35 @@ void skip_unimplemented_arg_scale(const attr_t &attr, res_t *res) {
     }
 }
 
+void skip_unimplemented_po_quantity(const attr_t &attr, res_t *res) {
+    // Generic SYCL backend currently cannot support port more than 5 post-ops.
+    if (is_generic_gpu()) {
+        if (attr.post_ops.len() > 5) {
+            res->state = SKIPPED;
+            res->reason = skip_reason::case_not_supported;
+            return;
+        }
+    }
+}
+
+void skip_unimplemented_attr(const attr_t &attr, res_t *res,
+        dnnl_primitive_kind_t prim_kind, dnnl_data_type_t src_dt,
+        dnnl_data_type_t dst_t) {
+
+    if (prim_kind != dnnl_binary && prim_kind != dnnl_group_normalization) {
```

Comment:
These `if`s shouldn't be here. It's just the unconditional list of function calls. Those functions should handle a configuration properly, thus, if there's no sum po, the function shouldn't do any checks.